src/riverroute/RtmHistFile.F90

```diff
@@ -983,8 +983,9 @@ subroutine RtmHistHtapesWrapup( rstwr, nlend )
     ! and write data to history files if end of history interval.
     do t = 1, ntapes
 
-       ! Skip nstep=0 if monthly average
-       if (nstep==0 .and. tape(t)%nhtfrq==0) cycle
+!KO    ! Should no longer be needed
```

Comment:
No need to leave commented code here.  Please remove.

Reply:
Done.